core/emulate.c

```diff
@@ -81,14 +81,14 @@
 
 /* Emulator ops */
 #define READ_GPR(idx, size) \
```

Comment:
The need for `READ_GPR` is very much deminished by this commit - consider removing it entirely.